test/integration/chat-board-state.test.js

```
@@ -70,6 +70,17 @@ class MockKanbn {
             )
         };
     }
+
+    async findTaskColumn(index, taskId) {
+        if (index && index.columns) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)